test/controllers/users_controller_test.rb

```diff
@@ -272,7 +272,7 @@ class UsersControllerTest < ActionController::TestCase
     @request.session.clear
     @request.env['HTTP_REMOTE_USER'] = 'ares'
     get :extlogin
-    assert_redirected_to edit_user_path(User.unscoped.find_by_login('ares'))
+    assert_redirected_to ApplicationHelper.current_hosts_path
```

Comment:
Do you think it is necessary to add a test to check if an external user with `mail_enabled` set to true is redirected to the edit user page after creation?

Reply:
No harm in adding the test, so I will do that.

Reply:
Added.